### ISSUE TITLE
randomize static encounters with a certain gen

### ISSUE THREAD

**Issue by michaelmuko** [kwsch/pkNX#376]
In pokemon legends arceus, I have seen the option to randomize area pokemon with certain gen pokemon only that I define, I wanted to know if it is possible to randomize static encounters with for example gen 1 pokemon only?

**Comment by Lusamine** [comment]
No, this wouldn't be very practical given the small dex. Feel free to submit a pull request if you can think of how to implement this reasonably.
